lib/components/map/connected-transitive-overlay.js

```diff
@@ -7,6 +7,12 @@ import { getTransitiveData } from '../../util/state'
 
 const mapStateToProps = (state, ownProps) => {
   const { labeledModes, styles } = state.otp.config.map.transitive || {}
+
+  // If trip viewer is not active, do not show trip on map
+  if (state.otp.ui.mainPanelContent !== null) {
+    return {tripData: null}
```

Comment:
Should be `transitiveData: null`

Reply:
This wasn't needed and is removed in f8a0ef0741d967b00932688787346fba5dcc53fc